### DISCUSSION TITLE
Do not upload releases as zip inside the repo

### DISCUSSION THREAD

**Discussion by IceflowRE** [Mogli12/KeyboardSteer#786] [Ideas]
In general not recommended to upload the release files as a zip and put it into the repository.
A great alternative would be to use a continuous integration and zip the files and place it into the release tab automatically.
This can happen as fixed releases with version numbers based on tags. 
And/ or a development release which will update itself automatically.

If you need help or are interested i can set it up for you and create a Pull Request.

**Comment by Mogli12** [comment]
I do not know. I put the ZIP file as "code" into the repository because I have to create it anyhow to test it locally. 

And real releases are published in ModHub.

**Reply by IceflowRE** [reply]
> I do not know. I put the ZIP file as "code" into the repository because I have to create it anyhow to test it locally.
> 
> And real releases are published in ModHub.

Thats all ok, but zips shouldn't be inside the repository, i mean you can, for local testing, still create it but the online one **here** can be done through a CI (for free).
I might setup an example the next days, and you can decide then.

**Comment by Mogli12** [comment]
The benefit of the ZIP in the repository is that it is versioned. It is easy for everybody to switch back to an old version. I do not want to loose this feature. But creating several released per day does not seem right for me as well.

**Reply by IceflowRE** [reply]
As i said, if you create a tag it will create a release with that number, which is never changed again and will remain [here](https://github.com/Mogli12/VehicleControlAddon/releases) forever and for every commit it might update a release named `development` which will be overriden of course.
Also if you really need a specific version again just download the repo and create the zip yourself tho.
